lib/Actions.js

```diff
@@ -76,6 +81,14 @@ Actions.prototype = {
         }, [selector], timeout, interval);
     },
 
+    /**
+     * @param {Function} evalFunction
+     * @param {Function} checkerFunction
+     * @param {array} [args]
```

Comment:
Also it would be great if you add some docs right in jsdoc, not only to readme, for users who prefer read code.